posthog/models/web_preaggregated/sql.py

```diff
@@ -435,6 +473,9 @@ def WEB_BOUNCES_INSERT_SQL(
                 argMinMerge(raw_sessions.initial_geoip_subdivision_city_name) AS city_name,
                 argMinMerge(raw_sessions.initial_geoip_subdivision_1_code) AS region_code,
                 argMinMerge(raw_sessions.initial_geoip_subdivision_1_name) AS region_name,
+                isNotNull(nullIf(nullIf(argMinMerge(raw_sessions.initial_gclid), 'null'), '')) AS has_gclid,
+                equals(nullIf(nullIf(argMinMerge(raw_sessions.initial_gad_source), 'null'), ''), '1') AS has_gad_source_paid_search,
```

Comment:
I guess technically you don't need the `nullIf` if you're just checking against the string `"1"`